src/box.tsx

```diff
@@ -36,7 +36,7 @@ Box.propTypes = {
   ...propTypes,
   innerRef: PropTypes.oneOfType([
     PropTypes.func,
-    PropTypes.shape({ current: PropTypes.element })
+    PropTypes.shape({ current: PropTypes.instanceOf(Element) })
```

Comment:
Won't this break server-side rendering if it requires `window`?

Reply:
@Rowno - I've updated this :)